src/main/java/org/testng/annotations/Parameters.java

```diff
@@ -14,7 +13,7 @@
  * @author <a href="mailto:cedric&#64;beust.com">Cedric Beust</a>
  */
 @Retention(RUNTIME)
-@Target({METHOD, CONSTRUCTOR, TYPE })
+@Target({METHOD, TYPE})
 public @interface Parameters {
```

Comment:
Sure.. But then again, I am not sure if this is something that needs to be deprecated.
Ping @cbeust @juherr - What do you suggest ?